### PR TITLE
moved to files, added pure builder

### PR DESCRIPTION
**Pull request by dibikhin**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,52 +1,56 @@
+/** 
+ * Entry point
+*/
+
+'use strict'
+
+const {
+   pipe,
+} = require('ramda')
```
Comment (dibikhin) [inline index.js]: `ramda` and `rambda` are diff modules

Comment (dibikhin) [inline index.js:9]: `ramda` and `rambda` are diff modules

Comment (vedamire) [reply index.js:9]: Lol, didn't notice

File: raptor-builder-fp.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ * @module RaptorBuilderFP_2
+ */
+
+'use strict'
+
+// NOTE can use `partial` here
+// NOTE new value last for forcing idempotence
+const withSpecimenId = specimenId => x => ({ ...x, specimenId, })
+const withSpeed = speed => x => ({ ...x, speed, })
+const withPlumage = plumage => x => ({ ...x, plumage, })
```
Comment (vedamire) [inline raptor-builder-fp.js:13]: 👍

**Review by vedamire — COMMENTED** [review]
Thanks for structuring code

**Pull request merged**